packages/trie/src/verifyRangeProof.ts

```diff
@@ -0,0 +1,500 @@
+import { nibblesToBuffer, nibblesCompare } from './util/nibbles'
+import { Trie } from './baseTrie'
```

Comment:
In fact, if you import `b.ts` in `a.ts` and import `a.ts` in `b.ts`, circular reference errors do not necessarily occur.
Nodejs will do some [processing](https://nodejs.org/api/modules.html#modules_cycles), first export an `unfinished copy` of a circular reference file.
The problem only occurs if these variables are used before the export is finished.
And here, there is no problem, because `verifyRangeProof.ts` exports only a function, the content of the function will not be executed when it is exported.